lib/routes/data/styles.js

```diff
@@ -63,4 +63,21 @@ module.exports = {
       ],
     ],
   },
+  breadcrumb: {
+    item: [
+      {
+        title: 'Dashboard',
```

Comment:
@scottnath - I had created this story for the data #577. Let me know if this is consumable.